00_prepare.sh

```diff
@@ -18,5 +13,5 @@ gpgcheck=0""" > /etc/yum.repos.d/racadm.repo
 dnf -y install openssl-devel srvadmin-idracadm7
 
 systemctl enable --now httpd
-kcli create host kvm -H 127.0.0.1 local
-kcli create pool -p /var/www/html default
+
+test -f /root/.ssh/id_rsa.pub || ssh-keygen -t rsa -N '' -f /root/.ssh/id_rsa
```

Comment:
for?

Reply:
this allows to deploy even when privtekey is set to False (which should be the default value since otherwise we re exposing the private key of the hypervisor in the vm, with no real benefit)

Reply:
:+1: